bin/Main.purs

```diff
@@ -151,13 +183,10 @@ main = launchAff_ do
             Right str ->
               Console.log str
 
-readOperatorTable :: Maybe FilePath -> Aff (Array String)
-readOperatorTable = case _ of
-  Nothing ->
-    pure defaultOperators
-  Just path -> do
-    table <- liftEffect <<< Buffer.toString UTF8 =<< FS.readFile path
-    pure $ String.split (Pattern "\n") table
+readOperatorTable :: FilePath -> Aff (Array String)
+readOperatorTable path
+  | path == ".tidyoperators.default" = pure defaultOperators
```

Comment:
I'm not totally sure how this is meant to work. Does this file exist?

Reply:
Oops, never mind, I see it being used as a default value in the worker config.

Reply:
Yeah. I guess I could use the empty string? I dunno, it's just a stub regardless because I can't pass `Maybe` values to workers 🤮.